frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -157,6 +155,10 @@ export default {
       this.payments.map(payment => {
         payment.checked = this.tableChecked
       })
+    },
+    dueDate (datems) {
+      const date = datems || new Date() // remote new Date() when dealing with real data.
```

Comment:
Can you enforce consistency here and everywhere else in the type usage?

I.e. either `humanDate` accepts milliseconds or a `Date` object, but not both.

So in this case, `new Date()` would be replaced with `Date.now()`, or even better, simply add a default value to the `humanDate` function (e.g. `humanDate(datems = Date.now()`, I think that might be possible, double-check me on that)

Reply:
The default value is a clean solution. Changed, as well as the other comments!